docker/pravega/scripts/init_kubernetes.sh

```diff
@@ -45,29 +45,37 @@ init_kubernetes() {
 
         local ns=${POD_NAMESPACE}
         local podname=${POD_NAME}
+        # SERVICE_NAME is set by pravega-operator when external access is enabled from version 0.5.0 onwards
+        # SERVICE_NAME and POD_NAME are different in pravega-operator 0.5.0 onwards 
+
+        if [ -z "${SERVICE_NAME}" ]; then
+          local servicename=${podname}
```

Comment:
Adding a new service_name variable breaks compatibility of operator with older pravega builds. Could we instead change the value of pod_name in operator 0.5 so it points to service name instead of pod name and hence no changes to pravega startup scripts are required?